python/graphstorm/eval/evaluator.py

```diff
@@ -1981,6 +1996,18 @@ def _get_major_score(self, score):
             major_score = score[self.major_etype]
         return major_score
 
+    def _get_early_stop_score(self, val_score: dict) -> float:
```

Comment:
Are there other classes that implement `GSgnnBaseEvaluator`? If yes they need to implement `_get_early_stop_score` as well.

Reply:
Yes. But most of they can use the GSgnnBaseEvaluator. _get_early_stop_score.